packages/react-app/src/components/Account.jsx

```diff
@@ -84,20 +84,24 @@ export default function Account({
 
   const { currentTheme } = useThemeSwitcher();
 
+  function isValidAddress (address) {
+    return address && address != "0x0000000000000000000000000000000000000000";
```

Comment:
Looks great and thanks for the new function.

I totally missed this but please consider using `!==` in place of `!=`